README.md

```diff
@@ -33,8 +33,12 @@ Documentation: [https://arduino-esp8266.readthedocs.io/en/2.4.1/](https://arduin
 ### Using git version
 [![Linux build status](https://travis-ci.org/esp8266/Arduino.svg)](https://travis-ci.org/esp8266/Arduino)
 
-- Install Arduino 1.8.2 from the [Arduino website](http://www.arduino.cc/en/main/software).
-- Go to Arduino directory
+- Install the current upstream Arduino IDE at the 1.8 level or later. The current version is at the [Arduino website](http://www.arduino.cc/en/main/software).
+- Go to Arduino application directory
+  - For Mac OS X normally: 
+  ```bash
+  cd /Applications/Arduino.app/Contents/Java
```

Comment:
Is it normal to install Arduino into global applications directory, instead of putting it somewhere in home directory?

Maybe we can relax this to `/path/to/Arduino.app`?

Reply:
I used to use mac too some time ago, and that was the place where I installed Arduino, so the place seemed right to me.
But you are right, not everyone is root on his machine, or want to install a single version of arduino in the global app dir.
I will fix this.

Reply:
Yes, you are right. In Mac OS X you can put the app bundle everywhere.
It seems there is no easy way to get the folder of an app.